components/MathTooltipContent.tsx

```
@@ -0,0 +1,90 @@
+export function MathTooltipContent({
+  content,
+  className,
+}: Readonly<MathTooltipContentProps>) {
+  const renderedContent = useMemo(() => {
+    return renderMathContent(content);
+  }, [content]);
+
+  return (
+    <div
+      className={className}
+      dangerouslySetInnerHTML={{ __html: renderedContent }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)